workflow/pred_temperature_ridge.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        rgdr.fit(x_train, y_train)
</pre><p> </p><p>I get this error message:</p><p> </p><pre class="ql-syntax" spellcheck="false">Exception has occurred: ValueError

One or more of the specified variables cannot be found in this dataset
  File "/Users/semvijverberg/surfdrive/Scripts/escience/cookbook/workflow/pred_temperature_ridge.py", line 184, in &lt;module&gt;
    rgdr.fit(x_train, y_train)
ValueError: One or more of the specified variables cannot be found in this dataset
</pre><p> </p><p>I tried to debug it, but somehow I could not go deeper that the main python script (pred_temperature_ridge.py). @Yang, could you test if this works with, I updated my s2spy version to 0.3.0. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Hi @semvijverberg, just tried again with `s2spy` version 0.3.0 and the older one (0.2.1), in both cases the notebook runs without any issue on my machine. This is a bit weird.

Maybe try to use the development version of it by `pip install -e path_to_s2spy` and test it again?

Reply:
Thanks for the hint.

I needed to update my xarray, it works now!